charts/newrelic-infrastructure/templates/controlplane/_enabled_helper.tpl

```diff
@@ -0,0 +1,62 @@
+{{/*
```

Comment:
I am not sure if the best approach is to make the chart gkeAutopilot aware. 
HAve you considered managing the changes by config/docs?

Example. In the guided install/docs asking if we are deploying for GKE autopitol and in that case add:
 - controlPlane.enabled=false

Reply:
Is it intended to be a new hidden parameter? There is no explanation of its behavior in the values.yaml or docs

Reply:
An other possibility is to add a values.yaml in the values and provide just a safe check:

Example 
```
if autopilot==true && ( priviledge==true || controlPlane.enable == true){
fail " with autopilot you should disable blablabla"
}
```

Reply:
@paologallinaharbur I was planning for a streamlined configuration to simplify New Relic integration for Autopilot (and other providers). However, it's not overly complex to configure individual settings either. Our plan is to automate manifest generation across all our clusters, so the method of configuration isn't a major concern for us. I'm open to whether we document the configuration in values or introduce a switch akin to gkeAutopilot.

Here is the library for newrelic.common.gkeAutopilot is configured in another [PR](https://github.com/newrelic/helm-charts/pull/1178/files#diff-f9d62a22d26fd59016e07cda0a450faa953e9574a5c867d77cb7fb0c8305a152). I can add some documentation to reference this if I get a green light on using the library.  

I introduced the [_enabled_helper.tpl](https://github.com/newrelic/nri-kubernetes/pull/916/files/f967a2789e9de19bf9e0620e3a4b6a734298f594#diff-06ccb5096b2deda174168410b2138c3225bbea1c7432195092e7f1e8dc6111a1) because it was generating a controlplane manifest even when the deployment was disabled, again this is not really an issue, but figured it could be properly disabled.

Let me know if you'd like to meet and discuss a path forward.  I am hoping this changes make it easier for consumer to deploy New Telic to Autopilot.